src/screens/Products.jsx

```diff
@@ -2,18 +2,60 @@ import Navbar from "../components/Nav/Navbar";
 import { useProduct } from "../context/product-context";
 import ProductListing from "../components/ProductListing/ProductListing";
 import Filters from "../components/Filters/Filters";
+import { useFilters } from "../context/filter-context";
 
 const ProductsScreen = () => {
-  const { state } = useProduct()
+  const {
+    state: { products, loading, error },
+  } = useProduct();
+  const {
+    productListState: { sortBy, byStock, byFastDelivery, byRating, byCategory },
+  } = useFilters();
+
+  const filterProducts = () => {
+    let filteredProducts = products;
+
+    if (sortBy) {
+      filteredProducts = filteredProducts.sort((a, b) =>
+        sortBy === "LOW_TO_HIGH" ? a.price - b.price : b.price - a.price
```

Comment:
It is sorting High_To_Low for default value, I would suggest you to return the unsorted data for default value of sortBy.